.github/workflows/ci.yml

```diff
@@ -11,17 +11,22 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-    - uses: actions/checkout@v1
+
+    - uses: actions/checkout@v3
+
     - name: Set up Python 3.8
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v4
       with:
         python-version: 3.8
+        cache: 'pip'
+        cache-dependency-path: |
+          **/requirements*.txt
 
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
-        pip install --ignore-installed .[test,dev]
+        pip install .[test,dev]
```

Comment:
--ignore-installed not required anymore ?

Reply:
No, this was actually the cause of a bug... I interpreted "--ignore-installed" to mean, "do not install things again if they exist with another version", but actually it means "install over things even if they already exist" :sweat_smile:

Reply:
I was also confused by the interpretation :p